pkg/kubelet/nodestatus/setters.go

```diff
@@ -354,6 +355,12 @@ func MachineInfo(nodeName string,
 				// node status.
 				node.Status.Capacity[v1.ResourceName(removedResource)] = *resource.NewQuantity(int64(0), resource.DecimalSI)
 			}
+
+			if info.SwapCapacity != 0 {
```

Comment:
Should this be gated with the FeatureGate?

Reply:
Almost certainly yes, and we will need api review anyway

Reply:
Thanks, added :+1: